api/python/quilt3/imports.py

```diff
@@ -41,9 +41,10 @@ def exec_module(cls, module):
 
         elif len(name_parts) == 3:  # e.g. module.__name__ == quilt3.data.foo
             namespace = name_parts[2]
+            registry_passed = PhysicalKey.from_url(registry)
```

Comment:
It looks like it's usually called `registry_parsed` in the codebase:
```suggestion
            registry_parsed = PhysicalKey.from_url(registry)
```

Reply:
You are right.